testutil/filefixture/file_fixtures.go

```diff
@@ -35,6 +35,8 @@ var Beta Fixture = Fixture{"Beta",
 	},
 }.defaults()
 
+var Beta_Hash string = "9GYDihlrhHQRNPV10lms35kogosBekjqJVYzTj0O5H-QJYTU7vf0YAgh3XBWKKBC"
```

Comment:
Not really, not as part of filefixtures anyway.  They're embedded in a few other tests, though.

And I'm dubious about this one, too.  What we really should have is...?  A map of IO system kinds to a handful of well known hashes?

*Most* of the tests do their own round-trip stuff so that they're not irritatingly fragile if we do perform a protocol update, but it's more an objective than a strictly followed rule.